### PR TITLE
Ugrading installer

### PR DESCRIPTION
**Pull request by ngodfraind**
Web installer is now functionnal from the prebuilt archive (I hope).

### CODE REVIEW

**Comment by purplefish32** [comment]
LGTM

File: main/web-installer/lib/Claroline/WebInstaller/Installer.php [hunk 1]
```diff
@@ -58,14 +58,20 @@ public function install()
             $kernel = new $this->kernelClass('prod', false);
             $kernel->boot();
             $container = $kernel->getContainer();
-            $this->launchInstaller($container, $output);
+            $sqlFile = $this->appDir.'/../claroline.sql';
+
+            if (realpath($sqlFile)) {
+                $output->writeln('Importing database from prebuilt sql file...');
+                $container->get('doctrine.orm.entity_manager')->getConnection()->exec(file_get_contents($sqlFile));
+                $container->get('doctrine.orm.entity_manager')->flush();
```
Comment (stefk) [inline main/web-installer/lib/Claroline/WebInstaller/Installer.php]: That line is superfluous (no entities are involved here)

File: main/web-installer/lib/Claroline/WebInstaller/Installer.php [hunk 2]
```diff
@@ -58,14 +58,20 @@ public function install()
             $kernel = new $this->kernelClass('prod', false);
             $kernel->boot();
             $container = $kernel->getContainer();
-            $this->launchInstaller($container, $output);
+            $sqlFile = $this->appDir.'/../claroline.sql';
+
+            if (realpath($sqlFile)) {
```
Comment (stefk) [inline main/web-installer/lib/Claroline/WebInstaller/Installer.php]: `file_exists` is more idiomatic

**Comment by ngodfraind** [comment]
LGTM

**Comment by purplefish32** [comment]
LGTM

**Pull request merged**